src/Keboola/GoogleAnalyticsExtractor/GoogleAnalytics/Client.php

```diff
@@ -195,6 +225,58 @@ private function processResponse($response, $query)
         return $processed;
     }
 
+    private function processResponseMCF($response, $query)
+    {
+        if (empty($response['rows'])) {
+            return null;
```

Comment:
prijde me pitomy, ze to vraci null a pritom tady https://github.com/keboola/google-analytics-extractor/blob/master/src/Keboola/GoogleAnalyticsExtractor/Extractor/Antisampling.php#L49 se ocekava, ze je to pole.
Uz to tak bylo teda i predtim a jako funguje to, ale je to imho casovana bomba.

Reply:
Jo, tiez som sa nad tym pozastavil, opravim to.